Library/ViewModels/PostCampaignCheckoutViewModel.swift

```diff
@@ -86,8 +86,9 @@ public class PostCampaignCheckoutViewModel: PostCampaignCheckoutViewModelType,
     .skipNil()
 
     let context = initialData.map(\.context)
+    let project = initialData.map(\.project)
     let checkoutId = initialData.map(\.checkoutId)
-    let baseReward = initialData.map(\.rewards).map(\.first)
+    let baseReward = initialData.map(\.rewards).map(\.first).skipNil()
```

Comment:
What is the case where the first reward is `nil`? I thought we always had at least one reward (i.e. the no-reward award.)